### ISSUE TITLE
Ability to specify which environment variables should affect cache key

### ISSUE THREAD

**Issue by aomarks** [google/wireit#93]

**Comment by aomarks** [comment]
Proposal for syntax:

```
"wireit": {
  "build": {
    "env": {
      "foo": "bar",
      "baz": {
        "external": true
      }
    }
  }
}
```

So foo will be set to bar for that script, and baz will be just added to the fingerprint, with the value being inherited  (as it already would be).

**Comment by justinfagnani** [comment]
btw, I found a nice workaround for my case. I put all my environment variables into a `.env` file that's read with the `dotenv` package. Then I depend on that in my files array.

It doesn't differentiate between env variable, so any change to the file will cause all scripts in the current graph that depend on it to re-run, but for me that works out well.

**Comment by aomarks** [comment]
This feature was released in 0.9.1: https://github.com/google/wireit#environment-variables
